client/dashboard/plugins/scheduled-updates/types.ts

```diff
@@ -0,0 +1,11 @@
+import type { Site } from '@automattic/api-core';
+
+export interface ScheduledUpdateRow {
+	id: string;
+	site: Site;
+	lastUpdate: number | null;
+	nextUpdate: number;
+	active: boolean;
+	schedule: 'daily' | 'weekly';
```

Comment:
I'm adding a `Frequency` type in https://github.com/Automattic/wp-calypso/pull/105808/files
Maybe add it here as well.

Reply:
Sorry, not "adding". It already exists, just bringing it over from the frequency-selection section.